docs/user-guide/install.md

```diff
@@ -6,18 +6,73 @@ vSphere Docker Volume Service is simple to install. It has zero configuration an
 
-Here is a demo show casing esxcli
+Log into ESXi host and download the [latest release](https://bintray.com/vmware/vDVS/VIB/_latestVersion) of vDVS driver VIB on ESXi and initiate the install by specifying the full path to the VIB.
 
-<script type="text/javascript" src="https://asciinema.org/a/80405.js" id="asciicast-80405" async></script>
+```
+# esxcli software vib install -v /tmp/vmware-esx-vmdkops-0.12.ccfc38f.vib
+Installation Result
+   Message: Operation finished successfully.
+   Reboot Required: false
+   VIBs Installed: VMWare_bootbank_esx-vmdkops-service_0.12.ccfc38f-0.0.1
+   VIBs Removed:
+   VIBs Skipped:
+```
 
 # Install on VM
 
-We currently package the service as a RPM and Deb package. This is to be able to start the service before Docker engine starts. We will also support Docker plugin framework once it is ready.
+**vDVS managed plugin**
+* **Prerequisite**: Docker 1.13/17.03 and above
 
-Here is a demo showcasing the install in a Photon OS VM.
+**vDVS managed plugin life cycle**
 
-<script type="text/javascript" src="https://asciinema.org/a/80412.js" id="asciicast-80412" async></script>
+* **Installation**
+```
+~# docker plugin install --grant-all-permissions --alias vsphere vmware/docker-volume-vsphere:latest
+latest: Pulling from vmware/docker-volume-vsphere
+f07d34084e57: Download complete
+Digest: sha256:e1028b8570f37f374e8987d1a5b418e3c591e2cad155cc3b750e5e5ac643fb31
+Status: Downloaded newer image for vmware/docker-volume-vsphere:latest
+Installed plugin vmware/docker-volume-vsphere:latest
+
+~# docker plugin ls
+ID                  NAME                DESCRIPTION                           ENABLED
+831fd45343af        vsphere:latest      VMWare vSphere Docker Volume plugin   true
+```
+
+* **Enabling/Disabling plugin**
+
+```
+~# docker plugin disable vsphere
+vsphere
+~# docker plugin ls
+ID                  NAME                DESCRIPTION                           ENABLED
+831fd45343af        vsphere:latest      VMWare vSphere Docker Volume plugin   false
+
+~# docker plugin enable vsphere
+ vsphere
+~# docker plugin ls
+ID                  NAME                DESCRIPTION                           ENABLED
+831fd45343af        vsphere:latest      VMWare vSphere Docker Volume plugin   true
+```
+
+* **Removing plugin**
+
+```
+~# docker plugin rm vsphere
+vsphere
+~# docker plugin ls
+ID                  NAME                DESCRIPTION         ENABLED
+```
+
+**vDVS DEB/RPM based installation**
+
+**Note** DEB/RPM packages will be deprecated going forward and will not be available.
+
+```
+- sudo dpkg -i <name>.deb # Ubuntu or deb based distros
+- sudo rpm -ivh <name>.rpm # Photon or rpm based distros
+```
```

Comment:
Could add a note that all current containers that are using vsphere volumes will continue to be able to run and that there is no impact (yes).

Also, one more good/must have is that Docker/containerd doesn't have to be restarted to install the plugin - meaning all containers stop and start (unless live-restore is in use).

Could also verify and mention behavior with swarm mode in use.